cloudsync/tests/test_provider.py

```diff
@@ -2449,4 +2460,4 @@ def test_paths_match(provider):
         assert provider.paths_match(s1, s2, True)
         assert provider.paths_match(s1.join(ABCD), s2.join(abcD_), True)
         assert provider.paths_match(s1.join(abcd), s2.join(abcd_), True)
-        assert not provider.paths_match(s1.join(ABCD_), s2.join(abcd), True)
\ No newline at end of file
+        assert not provider.paths_match(s1.join(ABCD_), s2.join(abcd), True)
```

Comment:
Can you add a comment why these paths shouldn't match?

Reply:
added comments, and cleaned up tests a bit.

also added pytest.ini to get rid of the "unknown marks" warnings.